docs/security/cosmosdb-jupyter-notebook-2021-08-12.md

```diff
@@ -0,0 +1,66 @@
+
+# Content
+This document describes the mitigation of a security issue described [here](https://msrc-blog.microsoft.com/2021/08/27/update-on-vulnerability-in-the-azure-cosmos-db-jupyter-notebook-feature/).
+
+# Prerequisite
+The Industrial-IoT Platform has been deployed as described in the [Industrial-IoT github repository](https://github.com/Azure/Industrial-IoT/blob/main/docs/deploy/readme.md).
+The steps below refers to a new full installation of the Industrial-IoT Platform.
+
+# Validate proper operation
+- Open the "Engineering Tool" of the Industrial-IoT Platform of the deployment.
```

Comment:
when Engineering Tool was deployed, otherwise please use OpenAPI (swagger) description.

Reply:
was not deployed?